src/obfl-specification.html

```diff
@@ -343,13 +343,56 @@ <h3 id="L5025">Auxiliary Flows</h3>
 of these containers, starting from 1 each time.</p>
 
 <h3 id="L5023">Simplex</h3>
-<p class="markup">In a sequence where duplex is not enabled, only the front side of sheets are counted and paginated. 
-The back side of sheets are neither counted nor paginated.</p>
+<p class="markup">In a sequence where duplex is not enabled, only the front side
+  of sheets are counted and paginated. The back side of sheets are neither
+  counted nor paginated.</p>
 
 <h3 id="L5024">Duplex</h3>
-<p class="markup">In a sequence where duplex is enabled, both sides of sheets are counted and paginated.
-However, if the last sheet of a duplex sequence only has content on the front side, 
-the back side is counted, but not <em>paginated</em>.</p>
+<p class="markup">In a sequence where duplex is enabled, both sides of sheets
+  are counted and paginated. However, if the last sheet of a duplex sequence
+  only has content on the front side, it depends on the values of the optional
+  attributes break-before and break-after on sequence, dynamic-sequence and
+  toc-sequence whether there is a back side, and if so, whether it is counted or
+  paginated. The default behavior is that there is a back side and it is counted
+  but not <em>paginated</em>.</p>
+
+<ins datetime="20200612T15:00:00">The last page of a sequence where duplex is
+enabled, may not be paginated and may not be counted.</ins>
+<p></p>
+
+<div class="markup">The just-mentioned break-before and break-after attributes
+  insert a break before or after the sequence, respectively. Depending on the
+  specific element, some of these values are supported:
+  <dl>
+    <dt>'volume'</dt>
+      <dd>continue in a new volume with or after this sequence,
+        respectively.</dd>
+    <dt>'sheet'</dt>
+      <dd>continue on a new right-hand page with or after this sequence,
+        respectively. If this results in a page without content, don't skip the
+        header or footer. This behavior is consistent with break-before="sheet"
+        on a block.</dd>
+    <dt>'page'</dt>
+      <dd>continue on a new page with or after this sequence, respectively,
+        which may be a left-hand page. If there is a transition from duplex to
+        simplex or visa versa, or the page dimensions don't match, the current
+        or new sequence needs to start on a right-hand page.</dd>
```

Comment:
Simplify to "insert a page break"?

Reply:
Done. With some extra explanation that this value can be used to produce media without blank pages.